### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by lukasmrtvy**
Interested in this simple Dockerfile ?
Would be also great to setup official docker hub repository ( https://docs.docker.com/docker-hub/builds/ ).

### CODE REVIEW

**Comment by fbartels** [comment]
You could make the resulting image even smaller by using https://github.com/halverneus/static-file-server as the base image.

**Comment by fhemberger** [comment]
@fbartels Dockerfile would be nice and short this way:

```dockerfile
FROM halverneus/static-file-server
COPY . /web
```

Web server is exposed by default on port 8080.

**Comment by bastienwirtz** [comment]
Hi,
Thanks all of you! I'll look into it & test it asap. 
@fbartels @fhemberger the static file server image looks nice but the initial proposition is still quite small and clean, and as it come straight from the official base alpine image, it removes a level of dependency, it's a solid choice for maintainability.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.11
+
+COPY ./ /www/
+
+ENV USER abc
```
Comment (fhemberger) [inline Dockerfile]: Nit: Could you please rename the user/group to `darkhttpd`?

Comment (bastienwirtz) [reply Dockerfile]: +1 That would be great!

**Comment by lukasmrtvy** [comment]
I can add s6 overlay for runtime uid,gid change , if You want to implement some crud to local config in the future to support proper volume handling..

File: Dockerfile (hunk 1 above)
Comment (lukasmrtvy) [reply Dockerfile]: done

**Comment by bastienwirtz** [comment]
Finally got to test it, looks good :+1: 
Thanks!

**Comment by calvinbui** [comment]
@bastienwirtz could you enable the Docker Hub builds?

**Pull request merged**